src/zoid/auth/component.jsx

```diff
@@ -123,9 +125,10 @@ export function getAuthComponent() : ZoidComponent<AuthPropsType> {
                     queryParam: true,
                     default:    () => FUNDING.PAYPAL
                 },
-                
+
                 onApprove: {
-                    type:     'function'
+                    type:     'function',
+                    alias:    'onAuthorize'
```

Comment:
Since this is a new component it would be better to not add this legacy alias.

Reply:
done